src/Raven.Client/Documents/Session/IDocumentSession.TimeSeriesFor.cs

```diff
@@ -16,7 +16,7 @@ public partial interface IDocumentSession
     {
         ISessionDocumentTimeSeries TimeSeriesFor(string documentId, string name);
         ISessionDocumentTimeSeries TimeSeriesFor(object entity, string name);
-        ISessionDocumentTypedTimeSeries<TValues> TimeSeriesFor<TValues>(object entity, string name) where TValues : TimeSeriesEntry;
-        ISessionDocumentTypedTimeSeries<TValues> TimeSeriesFor<TValues>(string documentId, string name) where TValues : TimeSeriesEntry;
+        ISessionDocumentTypedTimeSeries<TValues> TimeSeriesFor<TValues>(object entity) where TValues : TimeSeriesEntry;
```

Comment:
I'm wondering if this isn't 1 step too far with taking the name from the class. Thoughts?

Reply:
I did it, because it play nicely with the `Register` and you don't need to worry about the name at all.

Reply:
The problem is that I can have few TS with identical values but different name and now you are forcing me to create separate classes for them.

Reply:
you can use the other overload, or create another class.
But for the majority of the use cases, it is nice, I think.

Reply:
Then I think we should either support
```
ISessionDocumentTypedTimeSeries<TValues> TimeSeriesFor<TValues>(object entity, string name = null) where TValues : TimeSeriesEntry;
```

or `DisplayNameAttribute` on that class